Content.Server/_Goobstation/Heretic/Abilities/HereticAbilitySystem.cs

```diff
@@ -68,6 +71,7 @@ public sealed partial class HereticAbilitySystem : EntitySystem
     [Dependency] private readonly StationSystem _station = default!;
     [Dependency] private readonly IMapManager _mapMan = default!;
     [Dependency] private readonly IPrototypeManager _prot = default!;
+    [Dependency] private readonly IGameTiming _timing = default!;
```

Comment:
unused - this system has like a billion unused dependencies but im only calling out the ones that are diffs